resources/shellHelperFunctions.sh

```diff
@@ -230,12 +230,12 @@ write_download_info() {
 set_version ( ) {
   PRODUCT_PATH=$1
 
-  if [ $BUILD_NUMBER ]; then
+  if [ $VERSION ]; then
```

Comment:
Does `write_download_info()` need to be updated for version/tier?

Reply:
`write_download_info()` doesn't appear to be used *anywhere* in the repo. I think it is used in build configurations so don't want to change it yet.

Reply:
I seem to recall it getting called for the configs that resync files